### PR TITLE
[WIP] paramedic hardsuit, office and access (sprites needed)

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request

todo:
[done]paramed hardsuit
[]paramed office that contains said hardsuit
[done]and paramed access number so only paramed/cmo has access to it
[done]gives paramed and cmo paramed access

## Why It's Good For The Game

paramed RN is just a slightly inferior doctor so adding this will spice them up a bit+ makes recovering dead bodies from space easier, also hardsuits are cool

## Changelog
:cl:
add: paramedic office! parameds own spiecal place over all currently avaliable maps!
add: paramedic hardsuit! a hardsuit for paramedic duties!
add: paramedic access code! this is so only parameds and the cmo can access the paramed office
/:cl:

### CODE REVIEW

**Comment by ToasterBan** [comment]
GOD DAMNIT STOP BREAKING

**Comment by ToasterBan** [comment]
now I just need to get a spriter to do the sprites and map this stuff up

**Comment by Archanial** [comment]
Giving parameds hardsuits is a bad idea. Just ask someone to open EVA and I'm pretty sure CMO has access to it.

**Comment by ToasterBan** [comment]
> Giving parameds hardsuits is a bad idea. Just ask someone to open EVA and I'm pretty sure CMO has access to it.

parameds have Eva access, this is meant to be a superior alternative to EVAs

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -557,6 +557,19 @@
 	name = "chief medical officer's hardsuit"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/medical/cmo
 
+/obj/item/clothing/head/hemlet/space/hardsuit/medical/emt
+	name = "paramedic hardsuit helmet"
+	desc = "A special helmet that protects against hazardous, low pressure enviroments. Built with lightweight materials for extra comfort, but does not protect the eyes from intense light. "
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 10, "energy" = 10, "bomb" = 20, "rad" = 75, "fire" = 50, "acid" = 75, "stamina" = 0)
+	resistance_flags = ACID_PROOF | FIRE_PROOF
+
+obj/item/clothing/suit/space/hardsuit/medical/emt
```
Comment (Sinestia) [inline code/modules/clothing/spacesuits/hardsuit.dm]: he do be gooncoding tho

**Review by Sinestia — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/clothing/spacesuits/hardsuit.dm]:
```suggestion
/obj/item/clothing/suit/space/hardsuit/medical/emt
```
:weary:

**Comment by ike709** [comment]
Stale

**Pull request closed without merging**